db/migrations/sqls/20220427152150-ddc-metric-up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS ddc_metric
+(
+    id     SERIAL NOT NULL,
+    name   TEXT   NOT NULL,
+    nodeId BIGINT,
+    PRIMARY KEY (name, nodeId)
```

Comment:
Why the primary key is not the `id` field? Why it consist of two fields?

Reply:
because combination name + nodeId is a unique identifier for the metric, having rows with the same values doesn't make sense

Reply:
this id needed to join tables. ddc_metric_value has metricId field.

Reply:
I just suggested here to do it in the same way as it was done in the above `faucet_pkey` example to make sure that the application is in the same style. We could have here 2 indices:  primary for `id` + regular for `name`. It's also possible that we can fetch data by `id` field.

Reply:
at the moment i don't see cases when this `id` index will be needed. Create 2 indexes instead of 1 just to have the same style - not sure. What benefit to you see here?

Reply:
at the moment i don't see cases when this `id` index will be needed. Create 2 indexes instead of 1 just to have the same style - not sure. What benefit do you see here?

Reply:
@upalinski @Dmitriy1984Molchanov 
Folks, let's not write novels in comments.
Please, schedule a 5 mins call to discuss.